tests/unit/src/test/scala/tests/classFinder/ClassBreakpointSuite.scala

```diff
@@ -178,6 +178,6 @@ class ClassBreakpointSuite extends BaseClassFinderSuite {
       buffers.put(path, sourceText)
       val sym = classFinder.findClass(path, pos.toLsp.getStart())
       assert(sym.isDefined)
-      assertNoDiff(sym.get, expected)
+      assertNoDiff(sym.get.value, expected)
```

Comment:
I wonder if this suite is relevant anymore, since breakpoints are handled within the `debug-adapter-library`. We could probably move everything to one suite and remove the public methods

Reply:
So both `findClass` and `findTasty` methods can be removed and test suites can be removed too?